open_academy/views/view_sessions.xml

```diff
@@ -0,0 +1,8 @@
+<odoo>
+    <record id="action_menu_sessions" model="ir.actions.act_window" >
```

Comment:
Follow the [documentation](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action) for id names